adafruit_turtle.py

```diff
@@ -158,21 +162,21 @@ def __init__(self, display=None, scale=1):
                     "No display available. One must be provided."
                 ) from err
 
-        self._w = self._display.width
-        self._h = self._display.height
+        self._w: int = self._display.width
+        self._h: int = self._display.height
         self._x = self._w // (2 * scale)
         self._y = self._h // (2 * scale)
         self._speed = 6
-        self._heading = 0
+        self._heading: float = 0
         self._logomode = True
         self._fullcircle = 360.0
         self._degreesPerAU = 1.0
         self._angleOrient = 1
-        self._angleOffset = 0
+        self._angleOffset: float = 0
         self._bg_color = 0
 
-        self._splash = displayio.Group()
-        self._bgscale = 1
+        self._splash: Any = displayio.Group()
```

Comment:
I would think this would be `displayio.Group`, was there something that made you type it as `Any`?